### PR TITLE
Move to file scoped namespace project Core part 1

### PR DESCRIPTION
**Pull request by gmottajr**
Move to file scoped namespace project Core part 1

### CODE REVIEW

**Review by seantleonard — APPROVED** [review]

**Review by seantleonard — APPROVED** [review]
Thank you for making these changes! Looks good.

**Review by aaronburtle — APPROVED** [review]

File: src/Core/Authorization/AuthorizationResolver.cs [hunk 1]
```diff
@@ -15,696 +15,696 @@
...
-                    string role = permission.Role;
-                    RoleMetadata roleToOperation = new();
-                    EntityAction[] entityActions = permission.Actions;
-                    foreach (EntityAction entityAction in entityActions)
-                    {
-                        EntityActionOperation operation = entityAction.Action;
-                        OperationMetadata operationToColumn = new();
+                    EntityActionOperation operation = entityAction.Action;
+                    OperationMetadata operationToColumn = new();
+
```
Comment (Aniruddh25) [inline src/Core/Authorization/AuthorizationResolver.cs:269]:
```suggestion
```

**Review by Aniruddh25 — APPROVED** [review]

**Comment by Aniruddh25** [comment]
Thank you @gmottajr for this contributions! I went ahead to resolve conflicts with main and get this merged in.

File: src/Core/Authorization/AuthorizationResolver.cs [hunk 2]
```diff
@@ -15,696 +15,695 @@
...
+    }
 
-            return executionPermitted;
-        }
+    public bool IsStoredProcedureExecutionPermitted(string entityName, string roleName, SupportedHttpVerb httpVerb)
+    {
+        bool executionPermitted = EntityPermissionsMap.TryGetValue(entityName, out EntityMetadata? entityMetadata)
+            && entityMetadata is not null
+            && entityMetadata.RoleToOperationMap.TryGetValue(roleName, out _);
+
```
Comment (Aniruddh25) [inline src/Core/Authorization/AuthorizationResolver.cs:126]:
```suggestion

```

File: src/Core/Authorization/AuthorizationResolver.cs [hunk 3]
```diff
@@ -15,696 +15,695 @@
...
-        /// <param name="allowedExposedColumns">Set of fields exposed to user.</param>
-        /// <param name="entityName">Entity from request</param>
-        /// <param name="allowedDBColumns">Set of allowed backing field names.</param>
-        private static void PopulateAllowedExposedColumns(HashSet<string> allowedExposedColumns,
-            string entityName,
-            HashSet<string> allowedDBColumns,
-            ISqlMetadataProvider metadataProvider)
+        return operation is EntityActionOperation.All ? EntityAction.ValidPermissionOperations : new List<EntityActionOperation> { operation };
+    }
+
```
Comment (Aniruddh25) [inline src/Core/Authorization/AuthorizationResolver.cs:419]:
```suggestion

```

File: src/Core/Authorization/AuthorizationResolver.cs [hunk 4]
```diff
@@ -15,696 +15,695 @@
...
+    /// <summary>
+    /// From the given parameters, processes the included and excluded column permissions to output
+    /// a list of columns that are "allowed".
+    /// -- IncludedColumns minus ExcludedColumns == Allowed Columns
+    /// -- Does not yet account for either being wildcard (*).
+    /// </summary>
+    /// <param name="allowedExposedColumns">Set of fields exposed to user.</param>
+    /// <param name="entityName">Entity from request</param>
+    /// <param name="allowedDBColumns">Set of allowed backing field names.</param>
+    private static void PopulateAllowedExposedColumns(HashSet<string> allowedExposedColumns,
```
Comment (Aniruddh25) [inline src/Core/Authorization/AuthorizationResolver.cs:429]:
```suggestion
    private static void PopulateAllowedExposedColumns(HashSet<string> allowedExposedColumns,
```

**Review by Aniruddh25 — APPROVED** [review]
Fix formatting

File: src/Core/Authorization/AuthorizationResolver.cs [hunk 5]
```diff
@@ -15,696 +15,693 @@
...
-        /// </summary>
-        /// <param name="allowedExposedColumns">Set of fields exposed to user.</param>
-        /// <param name="entityName">Entity from request</param>
-        /// <param name="allowedDBColumns">Set of allowed backing field names.</param>
-        private static void PopulateAllowedExposedColumns(HashSet<string> allowedExposedColumns,
-            string entityName,
-            HashSet<string> allowedDBColumns,
-            ISqlMetadataProvider metadataProvider)
+        return operation is EntityActionOperation.All ? EntityAction.ValidPermissionOperations : new List<EntityActionOperation> { operation };
+    }
```
Comment (Aniruddh25) [inline src/Core/Authorization/AuthorizationResolver.cs:417]:
```suggestion
    }

```

File: src/Core/Models/PaginationMetadata.cs [hunk 6]
```diff
@@ -3,62 +3,61 @@
 
 using Azure.DataApiBuilder.Core.Resolvers;
 
-namespace Azure.DataApiBuilder.Core.Models
+namespace Azure.DataApiBuilder.Core.Models;
+
+
```
Comment (Aniruddh25) [inline src/Core/Models/PaginationMetadata.cs:8]:
```suggestion
```

File: src/Core/Configurations/IConfigValidator.cs [hunk 7]
```diff
@@ -1,18 +1,17 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace Azure.DataApiBuilder.Core.Configurations
-{
+namespace Azure.DataApiBuilder.Core.Configurations;
+
```
Comment (Aniruddh25) [inline src/Core/Configurations/IConfigValidator.cs:5]:
```suggestion
```

**Review by Aniruddh25 — APPROVED** [review]
Remove extra lines

**Pull request merged**